draft-parecki-oauth-client-id-metadata-document.md

```diff
@@ -180,7 +185,35 @@ client metadata document accepted by authorization servers implementing their
 specification, for instance, preventing the registration of confidential clients
 by requiring the `token_endpoint_auth_method` property be set to `"none"`.
 
-TBD: We may want a property such as `client_id_expires_at` for indicating that the client is ephemeral and not valid after a given timestamp, especially for documents issued by a service for development purposes.
+The response for a client metadata document MAY return the `Expires` header, as
+defined in  {{RFC9111}}, to indicate to an Authorization Server that the
+client metadata document has a limited lifetime, and consequently, so does the
+OAuth Client created from the document. If the client metadata document cannot
+be re-fetched by the Authorization Server after the `Expires` time, then the
+Authorization Server MUST treat that `client_id` as being an `unauthorized_client`.
+
+### Client Metadata Documents for Development Use {#documents_for_development}
+
+When developing applications against a service that use this internet draft,
+developers often encounter the issue of “how do I serve a client metadata document
+at a `https` URL whilst developing my application?”.
+
+For this purpose, it is recommended to use a service that generates and hosts client
+metadata documents for development purposes.
```

Comment:
I am really strongly encouraging this over the alternative of just sticking the document anywhere on the web with a public URL for development purposes.